pyproject.toml

```diff
@@ -33,6 +33,8 @@ dependencies = [
     "protobuf==3.20.2",
     "onnxscript==0.1.0.dev20240327",
     "sympy",
+    "networkx==3.1",
```

Comment:
These are not required, will be downloaded as part of torch.
also `torch==2.0.0` is not allowed, we need to explicitly pass `+cpu`

Reply:
Yes, it should but throwing error-
![image](https://github.com/user-attachments/assets/c5676122-644f-4575-8abc-63bcd3f086f1)

Reply:
Pls check the readme, these changes are not required, pls revert the PR.